tsdb/head_test.go

```diff
@@ -4494,7 +4494,7 @@ func generateBigTestHistograms(n int) []*histogram.Histogram {
 		}
 
 		for j := 0; j < numSpans; j++ {
-			s := histogram.Span{Offset: 1 + int32(i), Length: spanLength}
+			s := histogram.Span{Offset: 1, Length: spanLength}
```

Comment:
This change prevents a new chunk from being cut on every append, and I don't think it affects the benchmark that uses this method but let me know if I'm wrong about that!

Reply:
It's been a while since I wrote that, but from what I can remember yes, it looks like `j` makes more sense and I must have made a mistake using `i`.